packages/keychain/package.json

```diff
@@ -81,15 +90,5 @@
     "prettier": "^2.0.5",
     "triplesec": "^3.0.27",
     "zone-file": "^1.0.0"
-  },
-  "bugs": {
-    "url": "https://github.com/blockstack/blockstack.js/issues"
-  },
-  "sideEffects": false,
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "typings": "dist/keychain/src/index.d.ts",
```

Comment:
@kyranjamie typings here needs to exist. Wherever the types are built, we need to include that path

Reply:
@aulneau  That path doesn't exist, and a "types" equivalent properties does, that points to correct path

Reply:
`"types": "./dist/index.d.ts",`

Reply:
ah my bad!